racetime_bot/bot.py

```diff
@@ -106,7 +106,7 @@ def create_handler(self, race_data):
             extra_headers={
                 'Authorization': 'Bearer ' + self.access_token,
             },
-            ssl=self.ssl_context,
+            ssl=self.ssl_context if self.ssl_context is not None else True,
```

Comment:
Can you set this to `if self.ssl_context is not None else self.racetime_secure`? Otherwise I think it'll break when not using SSL (i.e. on a dev environment)

Reply:
I applied this change, do you have a dev environment setup you can use to test this? I suppose I could set up my own...